src/main/java/edu/wisc/my/messages/controller/MessagesController.java

```diff
@@ -31,6 +35,33 @@
                 response.setStatus(HttpServletResponse.SC_INTERNAL_SERVER_ERROR);
             }
         }
+
+    @RequestMapping(value="/datedMessages", method=RequestMethod.GET)    
+    public @ResponseBody void getDatedMessages(HttpServletRequest request,
+      HttpServletResponse response) {
+        response.setContentType("application/json");
+        JSONObject rawMessages = messagesService.getRawMessages();
+        JSONObject validMessages = new JSONObject();
```

Comment:
I don't think `valid` is the best way to characterize what we're filtering to here. Aren't they more `current` than they are `valid`?